torchaudio/datasets/utils.py

```diff
@@ -7,17 +7,18 @@
 import tarfile
 import threading
 import zipfile
-from io import TextIOWrapper
+from _io import TextIOWrapper
 from queue import Queue
 from typing import Any, Iterable, List, Optional, Tuple, Union
 
 import torch
 import urllib
+import urllib.request
 from torch.utils.data import Dataset
 from torch.utils.model_zoo import tqdm
 
 
-def unicode_csv_reader(unicode_csv_data: TextIOWrapper, **kwargs: Any) -> str:
+def unicode_csv_reader(unicode_csv_data: TextIOWrapper, **kwargs: Any) -> Any:
```

Comment:
This is not BC breaking yet since the inline typing isn't part of the release. `str` was added in [here](https://github.com/pytorch/audio/blame/master/torchaudio/datasets/utils.py#L20).

Reply:
This was changed due to `csv.reader` has as output Any instead of str so this function actually will give `Any` and not solely `str` has output